.github/workflows/presubmit.yml

```diff
@@ -133,9 +146,29 @@ jobs:
           set -x
           echo "ANDROID_SDK_ROOT=$HOME/Library/Android/sdk" >> $GITHUB_ENV
           echo "DIST_DIR=$HOME/dist" >> $GITHUB_ENV
-
+      - name: "Run gradle to check if any of its projects are affected by changes"
```

Comment:
Why not name this after the ./gradlew command it runs like the others?

suggest: `./gradlew findAffectedModules --outputFilePath=${{ github.workspace }}/affected.txt`

Reply:
just put `./gradlew findAffectedModules` w/o arguments since they might be very long.